plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendClient.java

```diff
@@ -0,0 +1,779 @@
+/*
```

Comment:
Please minimize the scope of the initial PR as much as possible. You can refer to Exasol PR: https://github.com/trinodb/trino/pull/16202

Reply:
I add the related code according to the Exasol PR.